tests/topic_store.test

```diff
@@ -1,7 +1,4 @@
 <launch>
-    <env name="TOPIC_STORE_LOCAL_DB" value="YES"/>
```

Comment:
We could just change this line to 


```bash
<group if="$(optenv JENKINS_URL 0)">
    <env name="TOPIC_STORE_LOCAL_DB" value="YES"/>
</group>
```

to only run system mongod when JENKINS_URL is set

Reply:
or even something like $USER == "jenkins"

Reply:
Yeah, both are good options, if the Github Actions work though, and it allows others to test their own branches and PR, I'm not entirely sure it would be worth keeping the Jenkins build (unless it is used for anything else such as building docker, pypi or debs builds/deployments, etc..)

Reply:
Sounds good, we can combe back to if we need jenkins at a later date. I think all it is used for at the moment is building the ros-melodic-topic-store package

Reply:
If you have properly working github actions, no need to keep the CI jenkins builds, in my view. So, unit tests don't have to be run on jenkins, and you can disable them with a PR to our config here when you want.
https://github.com/LCAS/rosdistro/blob/e57626f2fe8ff122aefb1a2c1e5182faadb14184/melodic/lcas-dist.yaml#L828-L829

However, please ensure that the release process still works on jenkins, but that is very independent anyway.